dsc/tests/dsc_reference.tests.ps1

```diff
@@ -2,8 +2,13 @@
 # Licensed under the MIT License.
 
 Describe 'Tests for config using reference function' {
-    It 'Reference works' {
-        $out = dsc config get -f $PSScriptRoot/../examples/reference.dsc.yaml | ConvertFrom-Json
+    It 'Reference works for <operation>' -TestCases @(
+        @{ operation = 'get' },
+        @{ operation = 'test' }
```

Comment:
are there any tests for reference with `set` and `export`?

Reply:
Not at this time, will add later